db/message.go

```diff
@@ -0,0 +1,82 @@
+package db
```

Comment:
more helper functions for the database, concerning how to handle messages sent on a channel